portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,10 +39,12 @@
   private class Comment{
     String comment;
```

Comment:
these fields can be marked 'final' since there's no reason to update them after the initial assignment